go/libkb/chain_link.go

```diff
@@ -924,41 +924,39 @@ func (c *ChainLink) Unpack(m MetaContext, trusted bool, selfUID keybase1.UID, pa
 
 	// IF we're loaded from *trusted* storage, like our local
 	// DB, then we can skip verification later
-	if trusted {
-		if b, err := jsonparserw.GetBoolean(packed, "sig_verified"); err == nil && b {
-			c.sigVerified = true
-			m.VLogf(VLog1, "| Link is marked as 'sig_verified'")
-			if ckidata, _, _, err := jsonparserw.Get(packed, "computed_key_infos"); err == nil {
-				if uerr := c.UnpackComputedKeyInfos(ckidata); uerr != nil {
-					m.Warning("Problem unpacking computed key infos: %s", uerr)
-				}
+	if b, err := jsonparserw.GetBoolean(packed, "sig_verified"); err == nil && b {
+		c.sigVerified = true
+		m.VLogf(VLog1, "| Link is marked as 'sig_verified'")
+		if ckidata, _, _, err := jsonparserw.Get(packed, "computed_key_infos"); err == nil {
+			if uerr := c.UnpackComputedKeyInfos(ckidata); uerr != nil {
+				m.Warning("Problem unpacking computed key infos: %s", uerr)
 			}
 		}
-		if b, err := jsonparserw.GetBoolean(packed, "hash_verified"); err == nil && b {
-			c.hashVerified = true
-		}
-		if b, err := jsonparserw.GetBoolean(packed, "chain_verified"); err == nil && b {
-			c.chainVerified = true
-		}
-		if b, err := jsonparserw.GetBoolean(packed, "payload_verified"); err == nil && b {
-			c.payloadVerified = true
-		}
-		if i, err := jsonparserw.GetInt(packed, "disk_version"); err == nil {
-			c.diskVersion = int(i)
-		}
+	}
+	if b, err := jsonparserw.GetBoolean(packed, "hash_verified"); err == nil && b {
+		c.hashVerified = true
+	}
+	if b, err := jsonparserw.GetBoolean(packed, "chain_verified"); err == nil && b {
+		c.chainVerified = true
+	}
+	if b, err := jsonparserw.GetBoolean(packed, "payload_verified"); err == nil && b {
+		c.payloadVerified = true
```

Comment:
are we saving to disk with any of the above being `false`? i think it's fine to leave this, just trying to make sure I understand

Reply:
right,  it's always `true`, was making minimal changes. thx!